katello.gemspec

```diff
@@ -44,12 +44,12 @@ Gem::Specification.new do |gem|
   gem.add_dependency "anemone"
 
   #pulp3
-  gem.add_dependency "pulpcore_client", ">= 3.0.0", "< 3.1.0"
-  gem.add_dependency "pulp_file_client", ">= 0.1.0", "< 0.2.0"
-  gem.add_dependency "pulp_ansible_client", "<= 0.2.0b7.dev01574717759"
-  gem.add_dependency "pulp_container_client", ">= 1.0.0", "< 1.1.0"
-  gem.add_dependency "pulp_rpm_client", "<= 3.1.0b1.dev01576187357"
-  gem.add_dependency "pulp_2to3_migration_client", "< 0.0.2"
+  gem.add_dependency "pulpcore_client", ">= 3.2.1", "< 3.3.0"
+  gem.add_dependency "pulp_file_client", ">= 0.2.0", "< 0.3.0"
+  gem.add_dependency "pulp_ansible_client", ">=0.2.0b11", "< 0.2.0b12.dev01584138469"
+  gem.add_dependency "pulp_container_client", ">= 1.2.0", "< 1.3.0"
+  gem.add_dependency "pulp_rpm_client", ">=3.2.0", "< 3.3.0"
+  gem.add_dependency "pulp_2to3_migration_client", ">= 0.2.0b1", "< 0.2.0b2.dev01585075322"
```

Comment:
can we change this to < 0.3 ?

Reply:
meaning, replace " "< 0.2.0b2.dev01585075322" with "< 0.3"

Reply:
That installed pulp_2to3_migration_client 0.2.0b2.dev01585506724 . We want to be on 0.2.0b1..That is the one we packaged. No?

Reply:
we dont' typically pin things that tightly to whats foreman-packaging, but instead update foreman-packaging based on whats in the gemspec (as new stuff becomes available).  However since they are going to revamp the migration quite a bit and break how we interact, i guess its fine to keep this for now.  Normally, if we weren't expecting breaking changes, i'd say stick with:    ">= 0.2"  "< 0.3"